packages/agent/src/connection-server.ts

```diff
@@ -0,0 +1,59 @@
+import { spawn, fork, Operation } from 'effection';
+import { Mailbox, readyResource } from '@bigtest/effection';
+import { express, Express, Socket } from '@bigtest/effection-express';
+
+interface AgentConnectionServerOptions {
+  port: number;
+  inbox: Mailbox;
+  delegate: Mailbox;
+}
+
+export class AgentConnectionServer {
```

Comment:
Can we please call this object `AgentHandler` it's just too good a pun to pass up, and it's also very descriptive, not often that this happens!